web/libs/editor/src/lib/AudioUltra/Waveform.ts

```diff
@@ -301,6 +302,16 @@ export class Waveform extends Events<WaveformEventTypes> {
   async load() {
     if (this.isDestroyed) return;
 
+    // Warn about incompatible features when decoder is "none"
+    if (this.params.decoderType === "none") {
```

Comment:
if we do declare the type we can replace all the checks for a specific string (associated with type for that)